packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -240,7 +240,7 @@ export class MLSService extends TypedEventEmitter<Events> {
    * @param conversationId Id of the parent conversation which subconversation we want to leave
    */
   public async leaveConferenceSubconversation(conversationId: QualifiedId): Promise<void> {
-    const subconversationGroupId = await this.getGroupIdFromConversationId(
+    const subconversationGroupId = subconversationGroupIdStore.getGroupId(
```

Comment:
We can access store directly here (this is what `getGroupIdFromConversationId` would do under the hood)